tests/storage/test_files.py

```diff
@@ -24,16 +24,17 @@ def test_returns_size(self, file, size):
 def test_files_hash_correctly():
     file1 = io.BytesIO(b"This is my first file")
     assert (
-        files.hashfile(file1) == "2f1b1b913ca382ad8f992ec6a18ecedfa2fcd8ff21b0a2227614a7bd94c23d2d"
```

Comment:
Strange commit message. Does this mean "make code conform to `ruff check`"? or something like that?

Reply:
Also, as per our atomic commits convention - this commit should be squashed into the one that introduces these linting errors.

Reply:
Squashed!